backend/src/matches/entities/match.entity.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  RelationId,
+  ManyToOne,
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from 'src/users/entities/user.entity';
+import { Exclude } from 'class-transformer';
+
+export enum MatchStatusType {
+  HOME = 'HOME',
+  AWAY = 'AWAY',
+  DRAW = 'DRAW',
+  ABORTED = 'ABORTED',
+  IN_PROGRESS = 'IN_PROGRESS',
+}
+
+@Entity()
+export class Match {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ type: 'timestamptz' })
```

Comment:
```suggestion
  @CreateDateColumn()
```

I would consider using `CreateDateColumn` which automatically fills the column with the current datetime when the data is inserted into the db